README.md

```diff
@@ -46,24 +42,9 @@ Fork or clone the main git repository:
 
 `git clone git@github.com:stanford-esrg/retina.git`
 
-Write your first Retina application (see [examples](https://github.com/stanford-esrg/retina/tree/main/examples)):
-```rust
-use retina_core::config::default_config;
-use retina_core::subscription::TlsHandshake;
-use retina_core::Runtime;
-use retina_filtergen::filter;
-
-#[filter("tls.sni ~ '^.*\\.com$'")]
-fn main() {
-    let cfg = default_config();
-    let callback = |tls: TlsHandshake| {
-        println!("{:?}", tls);
-    };
-    let mut runtime = Runtime::new(cfg, filter, callback).unwrap();
-    runtime.run();
-}
-```
-Writing a Retina application consists of defining subscriptions. A subscription is defined by (1) [writing a filter](https://stanford-esrg.github.io/retina/retina_filtergen/index.html) to describe what subset of network traffic you're interested in, (2) choosing [data types to subscribe to](https://stanford-esrg.github.io/retina/retina_core/subscription/index.html), and (3) defining a callback function that takes in a subscribable data type and performs operations on the filtered data. 
+Write your first Retina application (see [examples](https://github.com/stanford-esrg/retina/tree/main/examples) and [documentation](https://stanford-esrg.github.io/retina/retina_core/index.html)).
```

Comment:
Is there an _extremely_ basic small example that someone can run to help understand syntax and structure of a multi-subscription Retina application? It's hard to tell from the README description which example is best to start off with and evidently some are more complex than others.

Reply:
For instance, it may be helpful to include something that only subscribes to connection records (but written in multi-subscription format) that simply logs connection 4-tuples to a file.